### PR TITLE
a few details in combinat (designs)

### PR DESCRIPTION
**Pull request by fchapoton**
Cleaning up a few files in combinat/designs, in particular about missing empty lines.

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [x] The description explains in detail what this PR is about.

### CODE REVIEW

File: src/sage/combinat/designs/group_divisible_designs.py [hunk 1]
```diff
@@ -90,36 +91,31 @@ def group_divisible_design(v,K,G,existence=False,check=False):
...
-    if (len(G) == 1 and
-        len(K) == 1 and
-        G[0]+1 in K):
+    if len(G) == 1 == len(K) and G[0] + 1 in K:
         from .bibd import balanced_incomplete_block_design
         k = K[0]
         if existence:
-            return balanced_incomplete_block_design(v+1,k,existence=True)
-        BIBD = balanced_incomplete_block_design(v+1,k)
+            return balanced_incomplete_block_design(v + 1, k, existence=True)
```
Comment (dcoudert) [inline src/sage/combinat/designs/group_divisible_designs.py:98]: could be simplified to `return balanced_incomplete_block_design(v + 1, k, existence= existence)`, no ?

File: src/sage/combinat/designs/group_divisible_designs.py [hunk 2]
```diff
@@ -90,36 +91,31 @@ def group_divisible_design(v,K,G,existence=False,check=False):
...
         if existence:
             return True
-        return GDD_4_2(v//2,check=check)
+        return GDD_4_2(v // 2, check=check)
 
     # From a TD(k,g)
-    elif (len(G) == 1 and
-          len(K) == 1 and
-          K[0]*G[0] == v):
+    elif (len(G) == 1 == len(K) and K[0] * G[0] == v):
```
Comment (dcoudert) [inline src/sage/combinat/designs/group_divisible_designs.py]: parenthesis could be removed

File: src/sage/combinat/designs/group_divisible_designs.py [hunk 3]
```diff
@@ -286,16 +284,17 @@ def __init__(self, points, groups, blocks, G=None, K=None, lambd=1, check=True,
                                     check=False,
                                     **kwds)
 
-        if (groups is None or
-            (copy is False and self._point_to_index is None)):
+        if (groups is None or (copy is False and self._point_to_index is None)):
```
Comment (dcoudert) [inline src/sage/combinat/designs/group_divisible_designs.py]: Remove outer parenthesis ?

**Review by dcoudert — COMMENTED** [review]
some minor comments. Otherwise, LGTM.

File: src/sage/combinat/designs/group_divisible_designs.py (hunk 1 above)
Comment (fchapoton) [reply src/sage/combinat/designs/group_divisible_designs.py:98]: hmm, no. There no other return until many lines after.

Comment (dcoudert) [reply src/sage/combinat/designs/group_divisible_designs.py:98]: You are right.

**Review by dcoudert — APPROVED** [review]
LGTM.

**Pull request merged**